### PR TITLE
fix(webui): search crash on ' ' as a value #898

### PR DESCRIPTION
**Pull request by ashishsurana**
**Type:**

The following has been addressed in the PR:

*  Issue Fixed: [#898](https://github.com/verdaccio/verdaccio/issues/898)

**Description:**

Resolves the above issue where search crashes on entering ' ' (spaces) as a value

### CODE REVIEW

**Comment by ashishsurana** [comment]
@juanpicado sure. Will do in a while

File: test/unit/webui/modules/home.spec.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    const wrapper = shallow(<Home />);
+
+    wrapper.instance().handleSearchInput({
+      target: {
+        value: 'test query string one'
+      }
+    });
+    expect(wrapper.state('query')).toBe('test query string one'); // on value '   '
+
+    wrapper.instance().handleSearchInput({
```
Comment (ayusharma) [inline test/unit/webui/modules/home.spec.js]: Please create a single reference for `handleSearchInput` and remove repetition.

File: test/unit/webui/modules/home.spec.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    const props = {
+      text:"Test Loader Text"
+    };
+    const wrapper = shallow(<Loading {...props} />);
+    expect(wrapper.find('p').text()).toEqual(
+      'Test Loader Text'
+    );
+  });
+
+  it('should trim the search value', () => {
```
Comment (ayusharma) [inline test/unit/webui/modules/home.spec.js]: only write test for `handleSearchInput`

**Comment by ashishsurana** [comment]
@ayusharma I have done the above changes. Please review the same.

**Comment by ayusharma** [comment]
Thanks, @ashishsurana  for PR.

**Pull request merged**